package.json

```diff
@@ -64,9 +64,9 @@
     "devtools:redux": "remotedev --hostname=localhost --port=8000",
     "start:dev": "concurrently -k -n build,react,redux yarn:start yarn:devtools:react yarn:devtools:redux",
     "announce": "node development/announcer.js",
-    "storybook": "start-storybook -p 6006 -c .storybook -s ./app,./.storybook/images",
+    "storybook": "start-storybook -p 6006 -c .storybook",
     "storybook:test": "jest --config=./jest.stories.config.js",
-    "storybook:build": "build-storybook -c .storybook -o storybook-build -s ./app,./.storybook/images",
+    "storybook:build": "build-storybook -c .storybook -o storybook-build",
```

Comment:
Removing static directory flags in `package.json` in favour of `staticDirs` in `.storybook/main.js`
https://github.com/storybookjs/storybook/blob/next/MIGRATION.md#deprecated---static-dir-cli-flag